docs/deployment/frameworks/open-webui.md

```diff
@@ -3,27 +3,35 @@ title: Open WebUI
 ---
 [](){ #deployment-open-webui }
 
+[Open WebUI](https://github.com/open-webui/open-webui) is an extensible, feature-rich,
+and user-friendly self-hosted AI platform designed to operate entirely offline.
+It supports various LLM runners like Ollama and OpenAI-compatible APIs,
+with built-in RAG capabilities, making it a powerful AI deployment solution.
+
+To get started with Open WebUI using vLLM, follow these steps:
+
 1. Install the [Docker](https://docs.docker.com/engine/install/)
 
-2. Start the vLLM server with the supported chat completion model, e.g.
+2. Start the vLLM server with a supported chat completion model:
 
-```console
-vllm serve qwen/Qwen1.5-0.5B-Chat
-```
+   ```bash
+   vllm serve qwen/Qwen1.5-0.5B-Chat
+   ```
 
-1. Start the [Open WebUI](https://github.com/open-webui/open-webui) docker container (replace the vllm serve host and vllm serve port):
+3. Start the Open WebUI Docker container (replace `vllm-serve-host` and `vllm-serve-port` with your actual vLLM host and port):
 
-```console
-docker run -d -p 3000:8080 \
---name open-webui \
--v open-webui:/app/backend/data \
--e OPENAI_API_BASE_URL=http://<vllm serve host>:<vllm serve port>/v1 \
---restart always \
-ghcr.io/open-webui/open-webui:main
-```
+   ```bash
+   docker run -d \
+       --name open-webui \
+       -p 3000:8080 \
+       -v open-webui:/app/backend/data \
+       -e OPENAI_API_BASE_URL=http://<vllm-serve-host>:<vllm-serve-port>/v1 \
+       --restart always \
+       ghcr.io/open-webui/open-webui:main
+   ```
 
-1. Open it in the browser: <http://open-webui-host:3000/>
+4. Open it in the browser: <http://open-webui-host:3000/>
 
-On the top of the web page, you can see the model `qwen/Qwen1.5-0.5B-Chat`.
+At the top of the page, you should see the model `qwen/Qwen1.5-0.5B-Chat`.
 
 ![](../../assets/deployment/open_webui.png)
```

Comment:
Thanks for updating the docs! Just note that based on @mgoin 's idea for changing a new model like Qwen/Qwen3-0.6B, the image here should also be replaced.

Reply:
Yes, updated.